runtime-modules/working-group/src/benchmarking.rs

```diff
@@ -787,12 +788,34 @@ benchmarks_instance! {
             None
         );
 
-        let current_budget = BalanceOf::<T>::max_value();
-        WorkingGroup::<T, _>::set_budget(RawOrigin::Root.into(), current_budget).unwrap();
-    }: _ (RawOrigin::Signed(lead_id.clone()), lead_id.clone(), current_budget, None)
+        let amount = BalanceOf::<T>::from(10_000u32);
+        let budget = BalanceOf::<T>::from(100_000u32);
+        let block_no = 100u32;
+        let current_block = <T as frame_system::Config>::BlockNumber::from(1u32);
+        WorkingGroup::<T, _>::set_budget(RawOrigin::Root.into(), budget).unwrap();
+    }: _ (RawOrigin::Signed(lead_id.clone()), lead_id.clone(), amount, None)
+    verify {
+        assert_eq!(WorkingGroup::<T, I>::budget(), BalanceOf::<T>::from(90_000u32), "Budget not updated");
+        assert_last_event::<T, I>(RawEvent::BudgetSpending(lead_id, amount, None).into());
+    }
+
+    vested_spend_from_budget {
+        let (lead_id, _) = insert_a_worker::<T, I>(
+            OpeningType::Leader,
+            0,
+            None
+        );
+
+        let amount_vesting = VestingBalanceOf::<T>::from(10_000u32);
```

Comment:
`AmountLow` is erroring because it is less than vesting pallet configuration for `MinVestedTransfer`